packages/lab/src/badge/Badge.css

```
@@ -24,44 +6,35 @@
   position: relative;
```

I would replace this with 
```suggestion
  position: absolute;
```

---

does adding "position: absolute" not position the element relative to the DOM. So in this case would appear top right of the screen ?

---

No, that is `position: fixed`